clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/EcsSpec.java

```diff
@@ -208,6 +208,79 @@ public void createServerGroupOperationTest() throws IOException { // no response
     assertNotNull(response); // TODO inspect response contents
   }
 
+  @Test
+  public void getServiceDiscoveryRegistriesTest() {
+    // given
+    ProviderCache ecsCache = providerRegistry.getProviderCache(EcsProvider.NAME);
+    String testClusterName = "spinnaker-deployment-cluster";
+    String testNamespace = Keys.Namespace.SERVICE_DISCOVERY_REGISTRIES.ns;
+    String clusterKey = Keys.getClusterKey(ECS_ACCOUNT_NAME, TEST_REGION, testClusterName);
```

Comment:
cache keys are formulated differently per resource; use `Keys.getServiceDiscoveryRegistryKey()` here.

side note: I'm surprised this test works with wrong key, though we're setting the correct top level namespace on lines 228-229 so I guess that's sufficient to retrieve the data from the test URL. (but still shouldn't use wrong key format b/c its confusing)

Reply:
modification is done.